docs/_templates/autosummary/base.rst

```diff
@@ -0,0 +1,5 @@
+{{ objname | escape | underline }}
```

Comment:
I think we made it worse :P, this hsouldn't show up here. Maybe start with a fresh PR?

Reply:
ok, I'll try to just copy the changes to new branch, branched from fresh main and see if it helps